### PR TITLE
Localize Notifications View Extension Buttons

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

Localize buttons on `Notifications View Extension`. Previous to this PR, they were hard coded on view layer, now switch to use resource string.

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@DynamoDS/dynamo 

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/Notifications/NotificationsView.xaml [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-             xmlns:local="clr-namespace:Dynamo.Notifications"
-            xmlns:Logging="clr-namespace:Dynamo.Logging;assembly=DynamoCore"
-                xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-
+        xmlns:local="clr-namespace:Dynamo.Notifications"
+        xmlns:p="clr-namespace:Dynamo.Notifications.Properties"
```
Comment (QilongTang) [inline src/Notifications/NotificationsView.xaml:7]: Added this namespace for extracting resource string

File: src/Notifications/NotificationsView.xaml [hunk 2]
```diff
@@ -62,34 +61,40 @@
...
-                            <Button Margin="50,20,20,20" HorizontalAlignment="Right" Style="{DynamicResource ResourceKey=STextButton}" Click="OKButton_Click">OK</Button>
-                    </StackPanel>
-                    </Border>
-                    <StackPanel Margin="0,0,20,0">
-                        
+                                <Button Margin="20,20,20,20"
+                                        Style="{DynamicResource ResourceKey=STextButton}"
+                                        Tag="{Binding ElementName=Details}"
+                                        Click="ShowDetails_ButtonClick"
+                                        Content="{x:Static p:Resources.ButtonDetails}">
```
Comment (QilongTang) [inline src/Notifications/NotificationsView.xaml:79]: Switch to use resource string

**Review by mjkkirschner — APPROVED** [review]
It looks good, do you think it might effect any UIAutomation tests?

**Comment by QilongTang** [comment]
@mjkkirschner I checked we do not have UI automation test for notifications view extension yet. We should be good.

**Pull request merged**